cwf/gateway/integ_tests/gx_reauth_test.go

```diff
@@ -374,7 +390,13 @@ func TestGxReAuthWithMidSessionPolicyInstallAndRemoval(t *testing.T) {
 	tr.WaitForReAuthToProcess()
 
 	// Check ReAuth success
-	assert.Contains(t, raa.SessionId, "IMSI"+imsi)
+	WaitForReAuthToProcess := func() bool {
```

Comment:
this callback should have a factory function that creates an instance bound to the specific closure you care about, don't copypasta this everywhere

Reply:
Thanks for catching this :-). Was being lazy here. Will fix it.